tests/unit/aiplatform/test_models.py

```diff
@@ -1131,3 +1189,104 @@ def test_print_model_if_exception(self):
             repr(test_model)
             == f"{object.__repr__(test_model)} failed with {str(mock_exception)}"
         )
+
+    @pytest.mark.usefixtures("get_model_with_supported_export_formats_artifact")
```

Comment:
Please parameterize with both `sync=True` and  `sync=False` path on all these tests.

Reply:
Done